extracted/plugins/SocketPlugin/src/common/SocketPluginImpl.c

```diff
@@ -129,9 +129,10 @@ struct sockaddr_un
 #endif
 #endif /* !ACORN */
 
-/* Solaris sometimes fails to define this in netdb.h */
-#ifndef  MAXHOSTNAMELEN
-# define MAXHOSTNAMELEN	256
+/* Standardize this to a minimum of 256 characters */
```

Comment:
```suggestion
/* Standardize this to a minimum of 256 characters.
The Socket plugin uses this value as a limit for the FQDN (Fully Qualified Domain Name) length. This is very restrictive on Linux, as 64 characters are not enough to resolve some domain names. Making it 256 if
it's not defined as greater fixes this issue on Linux (and any other platform where this could happen). */
```

I propose to add the explanation in the comment :)

Reply:
I think you can integrate the sussgestion by clicking on a button?

Reply:
I will merge it and then update the comment, because I cannot modify the original branch.

Reply:
👍